.github/workflows/build-ffmpeg.yml

```diff
@@ -344,24 +344,18 @@ jobs:
           CCPREFIX: ${{ steps.cross.outputs.CCPREFIX }}
         run: |
           echo "::group::configure extra flags for cross compilation"
-          extra_configure=""
           if [[ ${{ matrix.arch }} != "x86_64" ]]; then
             if [[ ${{ matrix.os_type }} == "macos" ]]; then
               export CFLAGS="${{ steps.cross.outputs.CFLAGS }}"
               export CXXFLAGS="${{ steps.cross.outputs.CXXFLAGS }}"
               export LDFLAGS="${{ steps.cross.outputs.LDFLAGS }}"
             fi
-            extra_configure=$(cat <<- VAREOF
-              -DCMAKE_TOOLCHAIN_FILE=${{ steps.cross.outputs.CMAKE_TOOLCHAIN_FILE }}
-          VAREOF
-          )
           fi
-          echo "$extra_configure"
           echo "::endgroup::"
 
           echo "::group::configure"
           PATH="$root_path/bin:$PATH" cmake -G "${{ matrix.cmake_generator }}" \
-            $extra_configure \
+            -DCMAKE_TOOLCHAIN_FILE=${{ steps.cross.outputs.CMAKE_TOOLCHAIN_FILE }} \
```

Comment:
There's no toolchain file for x86_64, so this will result in something like

`-DCMAKE_TOOLCHAIN_FILE=${ROOT_PATH}/cmake/toolchain/${{ matrix.arch }}-${{ matrix.os_type }}/crosscompile.cmake` ... pointing to a non-existent file.

I'm not sure it's worth the effort of refactoring the build workflow... probably easier to combine all of this into a single `CMakeLists.txt` file

Reply:
It's not that. The job that sets `steps.cross.outputs.CMAKE_TOOLCHAIN_FILE` doesn't run for x86_64, so the value is blank. CMake would complain if it were a non-existent file. It just ignores a blank value.

Reply:
On closer inspection, it _is_ using Clang. It's Xcode's Clang rather than `/usr/bin/clang`, but I get the feeling these are one and the same. I therefore don't know why it's failing without seeing ffmpeg's config log, but I'll try using GCC since that's what it was using before. Failing that, I'll get it to output the config log.

Reply:
Well that worked. Quit while we're ahead?

Reply:
Yes, I think this is good for now. I will merge after next Sunshine release.